test/unit/integrations/sage_pay_form/sage_pay_form_helper_test.rb

```diff
@@ -24,7 +24,7 @@ def test_basic_helper_fields
     assert_equal 5, @helper.fields.size
     assert_field 'Vendor', 'cody@example.com'
     assert_field 'Amount', '5.00'
-    assert_field 'VendorTxCode', 'order-500'
+    assert_field 'VendorTxCode', 'order-500' + @helper.identifier
```

Comment:
We should have another test that `@helper.identifier` doesn't contain any non digit characters. You can do this with a regex.